frontend/src/navbar.tsx

```diff
@@ -1,4 +1,5 @@
 import { FilterMenu } from "./filter-menu";
+import config from "../lib/config-loader";
```

Comment:
Ahh! Do you think I should just stick to updating the `document.title` instead?

Reply:
It's clever though, but I don't think it's what we want. This may not be as simple as I expected cause we're on the client side and the html is just served as a static file that webpack injects its script tags into:
https://github.com/iFixit/pulldasher/blob/master/app.js#L48

Reply:
We could pass the value as part of the `token` response and have it included in the [`pageContext`](https://github.com/iFixit/pulldasher/blob/master/frontend/src/page-context.ts#L8)

https://github.com/iFixit/pulldasher/blob/master/controllers/main.js#L11-L18

It'd get there a little late, but it'd be safe and pretty simple.